ODM-12551 Write documentation on telemetry metrics (#121)
genestack/admin-docs: docs/home/other/telemetry.md

# Telemetry

> Our company collects telemetry data to improve service quality, ensure stability, and enhance security. We analyze technical parameters such as resource utilization, and anonymized feature usage data.

Reviewer: ```suggestion
Genestack collects telemetry data to improve service quality, ensure stability, and enhance security. Analysis of both technical parameters such as resource utilization, and product metrics like anonymized feature usage data helps to improve ODM trustworthy for the most crucial aspects of it's work.
```

> Telemetry helps us to:

Reviewer: ```suggestion
More specifically, telemetry helps to:
```

- detect and resolve issues more efficiently

- optimize system performance and scalability

- make informed decisions about product improvements

- enhance reliability and user experience.

> We do not collect personal information without user consent and fully comply with applicable data protection laws and standards.

Reviewer: ```suggestion
Genestack doesn't collect any personal information without user consent, complying with data protection regulations; refer to security documents if you need more information in this regard.
```

> 

Reviewer: Possibly we can provide some examples of technical metrics. System-wide, infrastructure-specific, language-specific.

Author: I'm not sure, there are too many of them

Reviewer: Technical metrics includes groups of metrics to control:
 - Hardware parameters, such as CPU and memory consumption, disk space, etc.
 - Infrastructure parameters, such as kubernetes-specific measurements.
 - Programming language-specific metrics to track JVM behaviour controlling heap size and other parameters.

Author: Added

> In addition to technical metrics, we collect anonymized product usage data to better understand how our services are used and to guide product development decisions. These metrics help us identify what brings value to users and where improvements are needed.

Reviewer: ```suggestion
In addition to technical metrics, anonymized product usage data is being collected. It helps to understand better which features are used and to guide product development decisions. These metrics help us identify what brings value to users and where improvements are needed. The list of collected metrics includes but not limited to:
```

> - `product_number_of_active_users` - Number of registered unique users with “Active” status

Reviewer: ```suggestion
- `product_number_of_active_users` - Number of registered unique not-disabled users
```

> - `product_number_of_attached_files` - Number of attachments by Data class

Reviewer: ```suggestion
- `product_number_of_attached_files` - Number of Study File attachments by Data class
```

> - `product_number_of_signal_groups` - Number of signal groups

Reviewer: ```suggestion
- `product_number_of_signal_groups` - Number of Tabular, Variants and Flow Cytometry groups
```

- `product_number_of_slp` - Number of Samples, Libraries, and Preparations

> - `product_number_of_slp_groups` - Number of signal groups by Data class

Reviewer: ```suggestion
- `product_number_of_slp_groups` - Number of Sample, Library, and Preparation groups by Data class
```

> - `product_number_of_studies` - Number of studies

Reviewer: ```suggestion
- `product_number_of_studies` - Number of Studies
```
